guides/v2.3/graphql/reference/quote-payment-method.md

```diff
@@ -0,0 +1,160 @@
+---
+group: graphql
+title: setPaymentMethodOnCart mutation
+---
+
+The `setPaymentMethodOnCart` mutation defines which payment method to apply to the cart. Magento GraphQL supports the following payment methods:
+
+Title | Code
+--- | ---
+Authorize.Net | `authorizenet_acceptjs`
+Bank Transfer Payment | `banktransfer`
+Cash on Delivery | `cashondelivery`
+Check / Money order | `checkmo`
+No Payment Information Required | `free`
+Purchase Order | `purchaseorder`
+
+Apply the `setPaymentMethodOnCart` mutation after setting the shipping address, shipping method, and after applying any discounts to the cart.
+
+## Syntax
+
+`mutation: {setPaymentMethodOnCart(input: SetPaymentMethodOnCartInput): SetPaymentMethodOnCartOutput}}`
+
+## Example usage
+
+### Authorize.Net
+
+The following example assigns the `authorizenet_acceptjs` payment method to the specified cart.
+
+**Request**
+
+```text
+mutation {
+  setPaymentMethodOnCart(input: {
+    cart_id: "lvdqOLzryManseE2artECZuPClxFgG1o"
```

Comment:
can be a place holder in the doc, rather than actual cart id

Reply:
I'm going to leave the cart_id value as is for now. Changing it to a variable here would mean changing it in every other quote example.